src/core/definitions.jl

```diff
@@ -13,15 +13,93 @@ const IS = InfrastructureSystems
 const MOI = MathOptInterface
 const MOIU = MathOptInterface.Utilities
 const PJ = ParameterJuMP
-const MOPFM = MathOptFormat.MOF.Model
+const MOPFM = MOI.FileFormats.Model
 const TS = TimeSeries
 
+const BUILD_SIMULATION_TIMER = TimerOutputs.TimerOutput()
+const RUN_SIMULATION_TIMER = TimerOutputs.TimerOutput()
+
 #Type Alias for JuMP and PJ containers
 const JuMPExpressionMatrix = Matrix{<:JuMP.AbstractJuMPScalar}
-const PGAE{V} = PJ.ParametrizedGenericAffExpr{Float64, V} where V<:JuMP.AbstractVariableRef
-const GAE{V} = JuMP.GenericAffExpr{Float64, V} where V<:JuMP.AbstractVariableRef
-const JuMPAffineExpressionArray = Matrix{GAE{V}} where V<:JuMP.AbstractVariableRef
-const JuMPAffineExpressionVector = Vector{GAE{V}} where V<:JuMP.AbstractVariableRef
+const PGAE{V} =
+    PJ.ParametrizedGenericAffExpr{Float64, V} where {V <: JuMP.AbstractVariableRef}
+const GAE{V} = JuMP.GenericAffExpr{Float64, V} where {V <: JuMP.AbstractVariableRef}
+const JuMPAffineExpressionArray = Matrix{GAE{V}} where {V <: JuMP.AbstractVariableRef}
+const JuMPAffineExpressionVector = Vector{GAE{V}} where {V <: JuMP.AbstractVariableRef}
 const JuMPConstraintArray = JuMP.Containers.DenseAxisArray{JuMP.ConstraintRef}
 const JuMPParamArray = JuMP.Containers.DenseAxisArray{PJ.ParameterRef}
 const DSDA = Dict{Symbol, JuMP.Containers.DenseAxisArray}
+
+# Tolerance of comparisons
+const ComparisonTolerance = 1.0e-10
```

Comment:
Two comments here:

1) This is a value and a constant. Convention in the code base is to use ALL_CAPS.
2) There are two types of tolerances that can be set for comparison, absolute and relative tolerance, and we tend to be using absolute tolerance.

I would rename this to the following: 

```julia
const ABSOLUTE_TOLERANCE = 1e-10
```